app/js/components/LabResultsList.jsx

```diff
@@ -351,19 +350,17 @@ export class LabResultsList extends PureComponent {
       errorMessage = "",
     } = selectedPatient;
 
-    const getPatientLabResults = () => {
+    const getPatientLabResults = () => 
       // build a list of all obs from the encounters that are of type LabSet or Test
-      return encounters.reduce((acc, encounter) => {
+      encounters.reduce((acc, encounter) => {
```

Comment:
Why the removal of the return here?  I haven't been following 100%, but is it no longer (or was it never) needed?

Reply:
This was part of the third commit, so was an automated change by eslint, as should be apparent from the commit names. Looks like it changed `() => { return x }` to `() => x`, which is semantically equivalent.